http/src/main/resources/init-resources/gce-init.sh

```diff
@@ -141,8 +141,8 @@ CERT_DIRECTORY='/var/certs'
 DOCKER_COMPOSE_FILES_DIRECTORY='/var/docker-compose-files'
 WORK_DIRECTORY='/mnt/disks/work'
 GSUTIL_CMD='docker run --rm -v /var:/var gcr.io/google-containers/toolbox:20200603-00 gsutil'
-GCLOUD_CMD='docker run --rm -v /var:/var gcr.io/google-containers/toolbox:20200603-00 gcloud'
-DOCKER_COMPOSE='docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -v /var:/var docker/compose:1.29.1'
+GCLOUD_CMD='docker run --rm -v /var:/var -v /home/.docker:/.docker gcr.io/google-containers/toolbox:20200603-00 gcloud'
+DOCKER_COMPOSE='docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -v /var:/var -v "$PWD:$PWD" -w="$PWD" cryptopants/docker-compose-gcr'
```

Comment:
For docker-compose, does `docker/compose:1.29.1'` not work?

Also I wonder if mounting `-v /home/.docker:/.docker` would work (similar to the gcloud cmd) instead of setting `WORKDIR`..

Reply:
I don't think ` /home/.docker:/.docker` would work because `docker` runs as root...and I didn't find a way to specifiy path to the credHelper file (note /root is not writable for cos image).

A bit more info about the [docker-compose-gcr image](https://hub.docker.com/r/cryptopants/docker-compose-gcr)

Reply:
Ahh I see, thanks for the pointer 👍